test_runner/regress/test_storage_controller.py

```diff
@@ -1742,42 +1760,88 @@ def storcon_cli(args):
     storcon_cli(["node-configure", "--node-id", "1", "--availability", "offline"])
     assert "Offline" in storcon_cli(["nodes"])[3]
 
+    # Restore node, verify status changes in CLI output
```

Comment:
This is necessary because we now have two nodes, so making one unavailable kicks off migrations